zeppelin-zengine/pom.xml

```diff
@@ -312,6 +308,12 @@
       </activation>
 
       <dependencies>
+        <dependency>
+          <groupId>com.amazonaws</groupId>
```

Comment:
Any specific reason this is moved from there to only `hadoop2` profile? Hope this does not cause any problem with `hadoop3` profile.

Reply:
Initially this dependency is common for hadoop2 & hadoop3 but for hadoop3 we have added `hadoop-aws` dependency which clone all the required aws, s3 related classes and there are some conflict between the multiple versions of jar so we have removed `aws-java-sdk-s3` but forgot to added it in hadoop2 and that was the main reason of failing CI builds.

Reply:
> Any specific reason this is moved from there to only `hadoop2` profile? Hope this does not cause any problem with `hadoop3` profile.

 Initially this dependency is common for hadoop2 & hadoop3 but for hadoop3 we have added `hadoop-aws` dependency which clone all the required aws, s3 related classes and there are some conflict between the multiple versions of jar so we have removed `aws-java-sdk-s3` but forgot to added it in hadoop2 and that was the main reason of failing CI builds.